### PR TITLE
document `shift_cues_earlier/_later(_small)` ControlObjects

### PR DESCRIPTION
**Pull request by Swiftb0y**

### CODE REVIEW

**Comment by Swiftb0y** [comment]
idk why the build failed. I can't reproduce locally. Should I rebase onto 2.4 since 2.3 doesn't get updated anyway?

File: source/chapters/appendix/mixxx_controls.rst [hunk 1]
```diff
@@ -752,6 +752,58 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
    .. versionadded:: 2.0.0
 
 
+.. mixxx:control:: [ChannelN],shift_cues_earlier
+                   [PreviewDeckN],shift_cues_earlier
+                   [SamplerN],shift_cues_earlier
+
+   :range: binary
+   :feedback: The :term:`hotcues <hotcue>` move left by 10ms.
```
Comment (ronso0) [inline source/chapters/appendix/mixxx_controls.rst:760]: Note that this moves **all** cues. IIRC the intention is to compensate for decoder offsets.
https://github.com/mixxxdj/mixxx/blob/9b31059fbacdd2d66bcf397b2354bb32c7ad1872/src/track/track.cpp#L978

**Comment by ronso0** [comment]
> Should I rebase onto 2.4 since 2.3 doesn't get updated anyway?

Yup!

**Comment by Swiftb0y** [comment]
done. please rereview @ronso0 ;)

**Comment by ronso0** [comment]
> Note that this moves all cues.

I meant it moves all cue points: hotcues, saved loops, main cue, intro, outro (but not the temp. loop).
(well, the GUI button in skin settings is labeled 'Hotcue Shift buttons' ...)

**Comment by Swiftb0y** [comment]
right... so what text would you prefer then? Can you make a suggestion?

File: source/chapters/appendix/mixxx_controls.rst [hunk 2]
```diff
@@ -820,6 +820,46 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
    .. versionadded:: 2.0.0
 
 
+.. mixxx:control:: [ChannelN],shift_cues_earlier
+                   [PreviewDeckN],shift_cues_earlier
+                   [SamplerN],shift_cues_earlier
+
+   :range: binary
+   :feedback: All :term:`hotcues <hotcue>` move left by 10ms.
```
Comment (ronso0) [inline source/chapters/appendix/mixxx_controls.rst:828]: how about this?
Only downside is that we can not link to :term:.
Not sure if we should link to djing_with_mixxx#intro-and-outro-cues AND :term:cue AND :term:hotcues ?
```suggestion
   :feedback: All cue markers move left by 10ms.
```

Comment (Swiftb0y) [reply source/chapters/appendix/mixxx_controls.rst:828]: we could also just add a glossary entry for "cue marker" which in turn links to those?

Comment (ronso0) [reply source/chapters/appendix/mixxx_controls.rst:828]: of course 🙄 I'd prefer that, too

Comment (Swiftb0y) [reply source/chapters/appendix/mixxx_controls.rst]: Done

**Comment by ronso0** [comment]
Perfect, thanks!

**Comment by Swiftb0y** [comment]
Thanks.

**Pull request merged**